### PR TITLE
Add database status

### PR DESCRIPTION
**Pull request by neelxie**
#### What does this PR do?
It adds database statuses to the database list and also refactors the component to use css modules.

#### Ticket
https://trello.com/c/H0o35TXr

#### Screnshots
Screenshot from 2021-05-11 19-26-15

### CODE REVIEW

**Comment by kawerewagaba** [comment]
we need to work on the alignment of the table items, as well as the loading indicator/spinner

**Review by kawerewagaba — CHANGES_REQUESTED** [review]
we need to work on the alignment of the table items, as well as the loading indicator/spinner

**Review by kawerewagaba — CHANGES_REQUESTED** [review]
let's work on these changes

File: src/components/DatabaseList/DatabaseList.module.css [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  text-transform: uppercase;
+  padding: 1rem;
+  color: #008ac1;
+  text-align: center;
+  grid-row: 1;
+}
+
+.DatabaseTableRow {
+  display: grid;
+  grid-auto-columns: 1fr;
```
Comment (kawerewagaba) [inline src/components/DatabaseList/DatabaseList.module.css:75]: let's remove this rule please

File: src/components/DatabaseList/DatabaseList.module.css [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  color: #000000;
+  grid-template-columns: (5, minmax(0, 1fr));
```
Comment (kawerewagaba) [inline src/components/DatabaseList/DatabaseList.module.css:77]: Screenshot 2021-05-20 at 16 44 49
let correct this as well: `grid-template-columns: repeat(5, minmax(0, 1fr));`
i believe u left out the `repeat` function

**Review by kawerewagaba — CHANGES_REQUESTED** [review]
just a couple of changes and we're good

File: src/components/DatabaseList/DatabaseList.module.css (hunk 1 above)
Comment (neelxie) [reply src/components/DatabaseList/DatabaseList.module.css:75]: @kawerewagaba This has been removed

File: src/components/DatabaseList/DatabaseList.module.css (hunk 2 above)
Comment (neelxie) [reply src/components/DatabaseList/DatabaseList.module.css:77]: oops I forgot.

**Comment by kawerewagaba** [comment]
Screenshot 2021-05-21 at 06 59 35
u broke this instead. gonna merge, but work on a fix

**Review by kawerewagaba — APPROVED** [review]

**Pull request merged**